lib/user.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var UserStore = require('fh-wfm-user/lib/user/store');
```

Comment:
Hmm, in hindsight we should probably figure a way to make that part reusable by custom stores